sendgrid/version.py

```diff
@@ -1,2 +1 @@
-version_info = (6, 1, 0)
```

Comment:
Not sure if clients used `version_info` at all.

Reply:
It was used to build `__version__` (don't know why though), which is used here: https://github.com/sendgrid/sendgrid-python/blob/master/setup.py#L21

Reply:
Ah, need to update this: https://github.com/sendgrid/sendgrid-python/blob/ea796fb14e6c397e156ba37f30d4fdb06675a61f/sendgrid/helpers/mail/email.py#L7

Reply:
@thinkingserious `sys.version_info` returns the separated _python_ version information. It seemed like it was only used to build the `__version__` number in `version.py`, per your previous comment.